wallets/rn_cli_wallet/src/modals/SessionProposalModal.tsx

```diff
@@ -91,33 +90,6 @@ export default function SessionProposalModal() {
     return chains;
   }, [requestedChains]);
 
-  // get required chains that are not supported by the wallet
-  const notSupportedChains = useMemo(() => {
```

Comment:
@ignaciosantise Is `notSupportedChains` something that is needed for debugging w3w? It seems we only use it for logging to console rn

Reply:
yes, they were just for debugging purposes